crates/sp-domains/src/transaction.rs

```diff
@@ -1,3 +1,8 @@
+#[cfg(not(feature = "std"))]
+pub use self::runtime_decl_for_PreValidationObjectApi::PreValidationObjectApi;
```

Comment:
>Why this conditional compilation and why re-export is needed at all?

Previously I attempted to declare another runtime API under `lib.rs`, there is a constraint to `sp_api::decl_runtime_apis` that one module can't have multiple runtime apis defined, nested module is needed and it didn't compile without this re-export if I recall correctly. Since it's now in another module not in `lib.rs` directly, I checked it now and it's indeed unnecessary. Thanks for the comment!

Yeah, and it's the first time that we have multiple runtime api defined in one crate,  for I think it semantically nicer to use a new API instead of extending `ExecutorApi` directly and didn't find a better place to hold it.